_data/simple-icons.json

```diff
@@ -233,7 +233,7 @@
         {
             "title": "Aerospike",
             "hex": "C41E25",
-            "source": "http://pages.aerospike.com/rs/aerospike/images/Acid_Whitepaper.pdf"
+            "source": "https://aerospike.com/search/?addsearch=logo"
```

Comment:
Not sure about this one. The PDF have a vector version at the last page. We should at least have a search where we can find a PDF with their logo as first result.